homeassistant/components/simplisafe/__init__.py

```diff
@@ -486,7 +491,10 @@ def _async_process_new_notifications(self, system):
 
     async def async_init(self):
         """Initialize the data class."""
-        asyncio.create_task(self.websocket.async_websocket_connect())
+        asyncio.create_task(self.websocket.async_connect())
+        self._hass.bus.async_listen(
+            EVENT_HOMEASSISTANT_STOP, self.websocket.async_disconnect()
```

Comment:
```suggestion
            EVENT_HOMEASSISTANT_STOP, self.websocket.async_disconnect
```

I'm pretty sure you don't need the `()`